### ISSUE TITLE
The `event_loop` bug is back on Windows

### ISSUE THREAD

**Issue by ThomasLaPiana** [ethyca/fides#1660]
### Bug Description

Random stack trace error after running any CLI command

#### Steps to Reproduce

1. Use windows
2. Install the latest version of fides (2.0.0)
3. run any CLI command

### Expected behavior

No random stack trace

### Screenshots

```
Exception ignored in: <function _ProactorBasePipeTransport.__del__ at 0x00000217417F9AF0>
Traceback (most recent call last):
  File "C:\Users\tlapiana\anaconda3\lib\asyncio\proactor_events.py", line 116, in __del__
    self.close()
  File "C:\Users\tlapiana\anaconda3\lib\asyncio\proactor_events.py", line 108, in close
    self._loop.call_soon(self._call_connection_lost, None)
  File "C:\Users\tlapiana\anaconda3\lib\asyncio\base_events.py", line 751, in call_soon
    self._check_closed()
  File "C:\Users\tlapiana\anaconda3\lib\asyncio\base_events.py", line 515, in _check_closed
    raise RuntimeError('Event loop is closed')
RuntimeError: Event loop is closed
```

### Environment

* Version: 2.0.0 (beta)
* OS: Windows 10
* Python Version: 3.9.12

### Additional context

This bug has been fixed once

**Comment by ThomasLaPiana** [comment]
@PSalant726 @seanpreston @sanders41 this got reintroduced with those `fideslog` changes I believe, but I wasn't close enough to them to know which ones

**Comment by PSalant726** [comment]
Welp. We took a mitigated risk and removed the helper that appeased Windows in #95. The Python docs claimed the issue was fixed in Python 3.8 but I guess they lied. I'll restore the helper tomorrow.

**Comment by PSalant726** [comment]
Blocked by ethyca/fideslog#96.
